lib/queue_bus/config.rb

```diff
@@ -46,6 +56,14 @@ def with_local_mode(mode)
       Thread.current.thread_variable_set(LOCAL_MODE_VAR, previous)
     end
 
+    def in_context(context)
```

Comment:
```suggestion
    # Overrides the current bus context (if any) for the duration of a block, adding a
    # `bus_context` attribute set to this value for all events published in this scope. 
    # This is a threadsafe implementation. After, the global setting will be resumed.
    def in_context(context)
```

Reply:
Thank you very much! This will also be helpful for when I add/update documentation in confluence!